recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -22,10 +23,18 @@ if "%CONDA_BLD_PATH%" == "" (
 )
 
 :: Increase pagefile size, cf. https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/155
+:: Both in the recipe and in the final package, this script is co-located with SetPageFileSize.ps1, see meta.yaml
+set ThisScriptsDirectory=%~dp0
+set EntryPointPath=%ThisScriptsDirectory%ExecuteSetPageFileSize.bat
```

Comment:
```suggestion
set EntryPointPath=%ThisScriptsDirectory%SetPageFileSize.ps1
```

Reply:
🤦
I've been travelling until today, trying to squeeze commits in here and there. Sorry for the annoying mistakes.